README.md

```diff
@@ -32,12 +32,13 @@ the `tests` directory should not be optimised.
 
 ## Generation
 
-See the [documentation for the generate.dart script in the src
-directory](src/README.md), which will will generate a number of
+See the [documentation for the generate.dart script in the `diagrams`
+directory](diagrams/README.md), which will will generate a number of
```

Comment:
you're inconsistent about whether this package should be named `diagram` or `diagrams`, which is leading to that issue with having that hack in the `pubspec.yaml`. I recommend auditing everything to make it all consistent, then you can remove the hack.

Reply:
Hmm. Actually, I'm not: the package is called "diagram" (in `packages/diagram`) and the location of the diagrams is in the `diagrams` directory, but that is confusing.  Perhaps I'll just rename the diagram package to "diagram_capture".

Reply:
Ah, my bad. I was very confused, either way. :-)
Current status seems better.